client/state/plans/selectors.js

```diff
@@ -53,3 +53,14 @@ export function getPlanRawPrice( state, productId, isMonthly = false ) {
 	}
 	return isMonthly ? parseFloat( ( plan.raw_price / 12 ).toFixed( 2 ) ) : plan.raw_price;
 }
+
+/**
+ * Returns the user currency code
+ * @param  {Object}  state     global state
+ * @param  {Number}  productId the plan productId
+ * @return {String}  currency code, ie 'USD'
+ */
+export function getCurrencyCode( state, productId ) {
+	const plan = getPlan( state, productId );
```

Comment:
In some selectors, we check for presence of the required second param, e.g.:

```js
if ( ! productId ) {
   return null;
}```

I noticed we don't do it much in `state/plans/selectors`. Why is that?